### PR TITLE
Privacy mode pages

### PR DESCRIPTION
**Pull request by caseyyee**
Partially addresses https://github.com/MozillaReality/FirefoxReality/issues/267

Will still need help from @keianhzo to hook up.

The styles are borrowed from https://github.com/MozillaReality/FirefoxReality/pull/170 with some slight changes.  Seems that DPI may have changed?  I needed to change default font sizing to something a little larger.

### CODE REVIEW

**Comment by keianhzo** [comment]
@caseyyee Looks good to me, just not sure about the "bookmarks and downloads" part as we don't currently have any. Maybe we should remove that part for the moment.
Other than that I'll wait for @thenadj review to merge and hook up about:privatebrowsing

File: app/src/main/assets/html/private.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            <p>When you browse in a Private Window. Firefox Reality <strong>does not save</strong>:</p>
+
+            <ul class="two-column">
+                <li>visited pages</li>
+                <li>cookies</li>
+                <li>searches</li>
+                <li>temporary files</li>
+            </ul>
+
+            <p>Firefox Reality <strong>will save</strong> your:</p>
```
Comment (keianhzo) [inline app/src/main/assets/html/private.html]: Don't currently have support for downloads and bookmarks. We should remove that part until we get that.

Comment (caseyyee) [reply app/src/main/assets/html/private.html]: OK cool.  Will do that.

**Comment by caseyyee** [comment]
@keianhzo 
Removed bookmarks and downloads bit.  Also bumped up the text size a little.

**Comment by thenadj** [comment]
Looks good to me based on the screen shot. Thx @caseyyee

**Comment by thenadj** [comment]
@caseyyee I checked the PB page in the latest build and there is a tiny little thing I'd like you to change. The two sets of bullets don't horizontally align correctly. I like the vertical set on the right which is a few pixels higher than the vertical set on the left. Match the left set so that they horizontally align and then it's ready. Thx!

**Pull request merged**